cleanlab/count.py

```diff
@@ -928,6 +947,11 @@ def estimate_cv_predicted_probabilities(
     clf_kwargs : dict, optional
       Optional keyword arguments to pass into `clf`'s ``fit()`` method.
 
+    validation_func : callable, optional
+      Optional callable function that takes two arguments, `X_val`, `y_val`.
```

Comment:
Would again replace docstring here with the shorter snippet above that just tells user to reference CleanLearning.fit

Reply:
done!